contents/tutorials/performance-metrics.md

```diff
@@ -0,0 +1,100 @@
+
+[![FCP happens in the second frame, as that's when the first text and image elements are rendered to the screen.](https://web-dev.imgix.net/image/admin/3UhlOxRc0j8Vc4DGd4dt.png?auto=format&w=1600)](https://web.dev/fcp/)
+*FCP happens in the second frame, as that's when the first text and image elements are rendered to the screen. [Source](https://web.dev/fcp)* 
+
+
+[According to Chrome team](https://web.dev/fcp/), a good FCP time is 1.8 seconds while anything above 3 seconds is considered poor. Factors that can affect FCP are:
+* Page Size and Complexity
+* Server Response Time ([Time to First Byte](https://web.dev/ttfb/)) 
+* JavaScript and other client-side scripts, particularly if they are resource-intensive and take a long time to execute.
+* Caching, since it can reduce the time it takes to download and render resources.
+*  Network Latency
+
+## Dom Interactive
+[Dom Interactive](https://developer.mozilla.org/en-US/docs/Web/API/Document/readyState) means that all HTML content on the page has been downloaded and parsed and is ready to be manipulated by JavaScript code. However, sub-resources such as scripts, images, stylesheets and frames are still loading. In general, DOM Interactive happens before FCP (since FCP requires the browser to have downloaded and rendered at least some content of the page). Any subsequent changes to the DOM beyond the inital page load also trigger DOM Interactive and thus it can be called multiple times.
+
+Factors affecting DOM Interactive are:
+* Page Size and Complexity. The larger and more complex a page is, the longer it takes for the browser to parse and process the HTML content, leading to a slower DOM Interactive time.
+* Number and Size of Scripts. The more scripts a page has and the larger their size, the longer it takes for the browser to download and execute them, which can delay DOM Interactive.
+* Server Response Time: Slow server response times can delay the delivery of the page's resources, including the HTML content, leading to slower DOM Interactive times.
+* Third-party Scripts.
+* Network Latency.
+
+## Page Loaded
+The whole page has [loaded](https://developer.mozilla.org/en-US/docs/Web/API/Window/load_event) and is ready for the user to interact with it, including all dependent resources such as stylesheets and images. 
+
+Note that this is not the same as [Time to Interactive](https://web.dev/tti/) (TTI), which is a more comprehensive metric. When the Page Loaded event is trigged, some page elements may still be loading or processing, and the page may not be fully responsive to user interactions. This is in contrast to TTI, which measures how long it takes to be able to interact with all the page's features and functionality (although Page Loaded is a big contributing factor to TTI).
```

Comment:
Slightly confused by this section - it's about Page Loaded but you spend most of your time talking about TTI?

Reply:
I agree that its not clear why we're talking about TTI. I'll rephrase this. As Andy mentioned, my goal was to give an idea of what a good benchmark could be. Also, when I read up on Page Loaded, my first question was "How similar is this to TTI", so I do think it's worth calling out the differences